### ISSUE TITLE
Narrator announces the expand/collapsed state of SplitButton control twice

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/microsoft-ui-xaml#1595]
GitHubTags: #A11y_OSvisualupdate, #A11yMAS, #A11yTCS, #Win32, #DesktopApp, #WindowsUILibrary, #MAS1.3.1, #Narrator

**Environment Details:**
Application Name: OS visual update
Windows Version: Windows10
Screen Reader: Narrator, NVDA 2019.2

**Steps to Reproduce:**
1. Launch Mini Control Gallery App from the installer location.
2. Screen with all generic controls would appear on left side of the screen.
3. Navigate to "Generic Controls" section.
4. Start narrator.
5. Navigate to the "SplitButton" control and listen if the expand collpase state is announced twice or not.

**Actual:**
When the screen reader users navigate to the “SplitButton” control for the first time the expand/collapse state is announced twice which is incorrect.

**Expected:**
The expand/collapse state should be announced just one for screen reader users on interacting with the “SplitButton” control.

**User Impact:**
If proper information is not announced to the screen reader users, then they will get confused.

**Attachment:**
[A11y_OSvisualupdate_MiniControGallery_GenericControls_Screenreader  The expandcollapse state is announced twice for “SplitButton” control using narrator.pptx](https://github.com/microsoft/microsoft-ui-xaml/files/3831170/A11y_ControlUIupdatebasedonSantoriniOSvisualupdate_MiniControGallery_GenericControls_Screenreader.The.expandcollapse.state.is.announced.twice.for.SplitButton.control.using.narrator.pptx)

**Comment by YuliKl** [comment]
Minor issue that repros in both MiniControlGallery and Xaml Controls Gallery.

**Comment by jevansaks** [comment]
Agreed, this is minor. It could also be a Narrator bug. We should check to see if there are any spurious events being fired in this scenario.

**Comment by chingucoding** [comment]
I can't seem to repro this with the latest release of the XAML Controls Gallery. I have tested this on insider build 19608, and the collapsed/expand state get's only announced once.

Maybe this was an issue with narrator which is fixed now, and we can close this issue?

**Comment by ranjeshj** [comment]
@MayuriNaik can you please activate this issue if you are still hitting it ? Thanks!
